option optimize_for = LITE_RUNTIME;
import "VisitRouteInfo.proto";
message VisitInfoAnswer
{
 required uint32 result = 1;
 repeated VisitRouteInfo routes = 2;
}
